lib/plugins/aws/utils/markovCluster.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
```

Comment:
Good question. We tried existing ones, but both were not able to compute distinct clusters (here's one commit where we tried one --> https://github.com/serverless/serverless/pull/3441/commits/f8d293576f00e148a22b88dd607c17c150f954c5).

This code is able to compute non-overlapping clusters.